Gems/LyShine/Code/Source/UiCanvasManager.cpp

```diff
@@ -808,94 +808,94 @@ UiCanvasComponent* UiCanvasManager::FindEditorCanvasComponentByPathname(const AZ
-    if (isCursorVisible)
-    {
-        // for some reason Unproject seems to work when given the viewport pos with (0,0) at the
-        // bottom left as opposed to the top left - even though that function specifically sets top left
-        // to (0,0).
-        const float viewportYInverted = cam.GetViewSurfaceZ() - viewportPos.GetY();
-
-        // Unproject to get the screen position in world space, use arbitrary Z that is within the depth range
-        Vec3 flippedViewportRayOrigin(viewportPos.GetX(), viewportYInverted, 0.f);
-        Vec3 flippedViewportRayForward(viewportPos.GetX(), viewportYInverted, 1.f);
-
-        cam.Unproject(flippedViewportRayOrigin, rayOrigin);
-
-        Vec3 unprojectedPosForward;
-        cam.Unproject(flippedViewportRayForward, unprojectedPosForward);
-
-        // We want a vector relative to the camera origin
-        Vec3 rayVec = unprojectedPosForward - rayOrigin;
-
-        // we want to ensure that the ray is a certain length so normalize it and scale it
-        rayVec.NormalizeSafe();
-        rayDirection = rayVec * rayLength;
-    }
-
-
-    AzFramework::EntityContextId gameContextId;
-    AzFramework::GameEntityContextRequestBus::BroadcastResult(gameContextId,
-        &AzFramework::GameEntityContextRequests::GetGameEntityContextId);
-
-    AzFramework::RenderGeometry::RayRequest request;
-    request.m_startWorldPosition = LYVec3ToAZVec3(rayOrigin);
-    request.m_endWorldPosition = LYVec3ToAZVec3(rayOrigin + rayDirection);
-
-    AzFramework::RenderGeometry::RayResult rayResult;
-    AzFramework::RenderGeometry::IntersectorBus::EventResult(rayResult, gameContextId,
-        &AzFramework::RenderGeometry::IntersectorInterface::RayIntersect, request);
-
-    if (rayResult)
-    {
-        AZ::EntityId hitEntity = rayResult.m_entityAndComponent.GetEntityId();
-        if (hitEntity.IsValid())
-        {
-            AZ::EntityId canvasEntityId;
-            UiCanvasRefBus::EventResult(canvasEntityId, hitEntity, &UiCanvasRefInterface::GetCanvas);
-            if (canvasEntityId.IsValid())
-            {
-                // Checkif the UI canvas referenced by the hit entity supports automatic input
-                bool doesCanvasSupportInput = false;
-                UiCanvasBus::EventResult(doesCanvasSupportInput, canvasEntityId, &UiCanvasInterface::GetIsPositionalInputSupported);
-
-                if (doesCanvasSupportInput)
-                {
-                    // set the hit details to the hit entity, it will convert into canvas coords and send to canvas
-                    bool handled = false;
-                    UiCanvasOnMeshBus::EventResult(handled, hitEntity,
-                        &UiCanvasOnMeshInterface::ProcessHitInputEvent, inputSnapshot, rayResult);
-
-                    if (handled)
-                    {
-                        return true;
-                    }
-                }
-            }
-        }
-    }
-
-
-    return false;
+    //const CCamera cam;
```

Comment:
@bosnichd Can you port over LYN-3680 to github so that the community knows about and can potentially fix it

Also does anyone from @o3de/sig-presentation  know if it is possible to get the query camera position now?